### PR TITLE
Updated descriptions and conflicts in all composer.jsons

### PR DESCRIPTION
**Pull request by aschempp**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -130,7 +131,6 @@
         "symfony/doctrine-bridge": "<3.4",
         "symfony/security-core": "4.2.* <4.2.3",
         "symfony/swiftmailer-bundle": "2.6.* <2.6.2",
-        "symfony/twig-bundle": "4.1.0",
```
Comment (leofeyer) [inline composer.json:133]: Why did you remove this?

File: core-bundle/composer.json [hunk 2]
```diff
@@ -106,8 +107,7 @@
         "lexik/maintenance-bundle": "2.1.4",
         "symfony/doctrine-bridge": "<3.4",
         "symfony/security-core": "4.2.* <4.2.3",
-        "symfony/swiftmailer-bundle": "2.6.* <2.6.2",
-        "symfony/twig-bundle": "4.1.0"
```
Comment (leofeyer) [inline core-bundle/composer.json]: Why?

File: composer.json (hunk 1 above)
Comment (aschempp) [reply composer.json:133]: because we're requiring `^4.2 || ^4.3` 😉

File: core-bundle/composer.json [hunk 3]
```diff
@@ -1,7 +1,8 @@
 {
     "name": "contao/core-bundle",
     "type": "symfony-bundle",
-    "description": "Contao 4 core bundle",
+    "description": "Provides the core functionality of Contao Open Source CMS.",
```
Comment (leofeyer) [inline core-bundle/composer.json]:
```suggestion
    "description": "Provides the Contao core functionality",
```

File: calendar-bundle/composer.json [hunk 4]
```diff
@@ -2,6 +2,7 @@
     "name": "contao/calendar-bundle",
     "type": "contao-bundle",
     "description": "Adds calendar functionality to Contao 4",
```
Comment (leofeyer) [inline calendar-bundle/composer.json]:
```suggestion
     "description": "Adds calendar functionality to Contao",
```

File: comments-bundle/composer.json [hunk 5]
```diff
@@ -2,6 +2,7 @@
     "name": "contao/comments-bundle",
     "type": "contao-bundle",
     "description": "Adds comments functionality to Contao 4",
```
Comment (leofeyer) [inline comments-bundle/composer.json]:
```suggestion
     "description": "Adds comments functionality to Contao",
```

File: faq-bundle/composer.json [hunk 6]
```diff
@@ -2,6 +2,7 @@
     "name": "contao/faq-bundle",
     "type": "contao-bundle",
     "description": "Adds FAQ functionality to Contao 4",
```
Comment (leofeyer) [inline faq-bundle/composer.json]:
```suggestion
     "description": "Adds FAQ functionality to Contao",
```

File: installation-bundle/composer.json [hunk 7]
```diff
@@ -2,6 +2,7 @@
     "name": "contao/installation-bundle",
     "type": "contao-bundle",
     "description": "Required to install and update Contao 4",
```
Comment (leofeyer) [inline installation-bundle/composer.json]:
```suggestion
     "description": "Required to install and update Contao",
```

File: listing-bundle/composer.json [hunk 8]
```diff
@@ -2,6 +2,7 @@
     "name": "contao/listing-bundle",
     "type": "contao-bundle",
     "description": "Allows to list arbitrary data in the Contao 4 front end",
```
Comment (leofeyer) [inline listing-bundle/composer.json]:
```suggestion
     "description": "Allows to list arbitrary data in the Contao front end",
```

File: manager-bundle/composer.json [hunk 9]
```diff
@@ -1,12 +1,17 @@
 {
     "name": "contao/manager-bundle",
     "type": "symfony-bundle",
-    "description": "Contao 4 manager bundle",
+    "description": "This bundle contains code necessary to run the Contao 4 Managed Edition.",
```
Comment (leofeyer) [inline manager-bundle/composer.json]:
```suggestion
    "description": "Integrates Contao into the Contao Managed Edition",
```

File: news-bundle/composer.json [hunk 10]
```diff
@@ -2,6 +2,7 @@
     "name": "contao/news-bundle",
     "type": "contao-bundle",
     "description": "Adds news functionality to Contao 4",
```
Comment (leofeyer) [inline news-bundle/composer.json]:
```suggestion
     "description": "Adds news functionality to Contao",
```

File: newsletter-bundle/composer.json [hunk 11]
```diff
@@ -2,6 +2,7 @@
     "name": "contao/newsletter-bundle",
     "type": "contao-bundle",
     "description": "Adds newsletter functionality to Contao 4",
```
Comment (leofeyer) [inline newsletter-bundle/composer.json]:
```suggestion
     "description": "Adds newsletter functionality to Contao",
```

**Review by leofeyer — CHANGES_REQUESTED** [review]

File: manager-bundle/composer.json (hunk 9 above)
Comment (aschempp) [reply manager-bundle/composer.json]: "Managed Edition" is a name to me, and should therefore be uppercase?

Comment (leofeyer) [reply manager-bundle/composer.json]: I don't think so. Then "Contao Core Bundle" would be a name, too, but we are writing it as "Contao core bundle".

Comment (aschempp) [reply manager-bundle/composer.json]: I think both would be fine for the core bundle. But "Contao Managed Edition" is a name for sure. 
https://duckduckgo.com/?q=contao+managed+edition
https://contao.org/de/news/release-der-contao-managed-edition.html

Comment (leofeyer) [reply manager-bundle/composer.json]: Almost all of the results are in German. Of course it is written "Contao Managed-Edition" there. 😄 



This is an English result (and also the official repo) and it clearly says "Contao 4 managed edition".

Comment (aschempp) [reply manager-bundle/composer.json]: so it should be different in english and german?

**Comment by aschempp** [comment]
I have also updated the authors of `contao/manager-bundle`, because that's what people will see in the Contao Manager as whom Contao is from.

File: calendar-bundle/composer.json [hunk 12]
```diff
@@ -1,7 +1,8 @@
 {
     "name": "contao/calendar-bundle",
     "type": "contao-bundle",
-    "description": "Adds calendar functionality to Contao 4",
+     "description": "Adds calendar functionality to Contao",
```
Comment (leofeyer) [inline calendar-bundle/composer.json]: This line seems to be indented too much.

File: manager-bundle/composer.json (hunk 9 above)
Comment (leofeyer) [reply manager-bundle/composer.json]: I have updated my suggestion based on https://github.com/contao/package-metadata/pull/123#issuecomment-537843102

**Review by leofeyer — CHANGES_REQUESTED** [review]
The changed "description" lines are all indented too much, otherwise 👍
